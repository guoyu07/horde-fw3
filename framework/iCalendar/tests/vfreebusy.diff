002- string(334) "BEGIN:VFREEBUSY
002+ 
003- ORGANIZER;CN=GunnarWrobel:MAILTO:wrobel@demo2.pardus.de
003+ Fatal error: Call to undefined method String::wordwrap() in /home/wrobel/usr/devel/gh/horde-glue/horde-fw3/framework/iCalendar/iCalendar.php on line 1055
004- DTSTAMP:20061122T230929Z
005- DTSTART:20061122T230000Z
006- DTEND:20070121T230000Z
007- FREEBUSY;X-UID=MmZlNWU3NDRmMGFjNjZkNjRjZjFkZmFmYTE4NGFiZTQ=;X-SUMMARY=dGVzd
008-  HRlcm1pbg==:20061123T050000Z/20061123T060000Z
009- FREEBUSY:20061123T080000Z/20061123T100000Z
010- END:VFREEBUSY
011- "
012- string(12) "GunnarWrobel"
013- string(22) "wrobel@demo2.pardus.de"
014- array(2) {
015-   [1164258000]=>
016-   int(1164261600)
017-   [1164268800]=>
018-   int(1164276000)
019- }
020- string(10) "testtermin"
021- array(1) {
022-   [1164261600]=>
023-   int(1164268800)
024- }
025- int(1164236400)
026- int(1169420400)
027- array(2) {
028-   [1164258000]=>
029-   int(1164261600)
030-   [1164268800]=>
031-   int(1164276000)
032- }
033- array(4) {
034-   [1164258000]=>
035-   int(1164261600)
036-   [1164268800]=>
037-   int(1164276000)
038-   [1164279600]=>
039-   int(1164283200)
040-   [1164290400]=>
041-   int(1164297600)
042- }
043- array(4) {
044-   [1164258000]=>
045-   array(2) {
046-     ["X-UID"]=>
047-     string(44) "MmZlNWU3NDRmMGFjNjZkNjRjZjFkZmFmYTE4NGFiZTQ="
048-     ["X-SUMMARY"]=>
049-     string(16) "dGVzdHRlcm1pbg=="
050-   }
051-   [1164268800]=>
052-   array(0) {
053-   }
054-   [1164279600]=>
055-   array(0) {
056-   }
057-   [1164290400]=>
058-   array(1) {
059-     ["X-SUMMARY"]=>
060-     string(8) "dGVzdA=="
061-   }
062- }
063- array(9) {
064-   [1164254100]=>
065-   int(1164254700)
066-   [1164254400]=>
067-   int(1164258300)
068-   [1164261300]=>
069-   int(1164261900)
070-   [1164268500]=>
071-   int(1164269100)
072-   [1164275700]=>
073-   int(1164276300)
074-   [1164279600]=>
075-   int(1164294000)
076-   [1164290400]=>
077-   int(1164290520)
078-   [1164292200]=>
079-   int(1164292500)
080-   [1164297300]=>
081-   int(1164297600)
082- }
083- int(1004297300)
084- int(1169420400)
085- array(11) {
086-   [1164258000]=>
087-   int(1164261600)
088-   [1164268800]=>
089-   int(1164276000)
090-   [1164279600]=>
091-   int(1164294000)
092-   [1164290400]=>
093-   int(1164297600)
094-   [1164254100]=>
095-   int(1164254700)
096-   [1164254400]=>
097-   int(1164258300)
098-   [1164261300]=>
099-   int(1164261900)
100-   [1164268500]=>
101-   int(1164269100)
102-   [1164275700]=>
103-   int(1164276300)
104-   [1164292200]=>
105-   int(1164292500)
106-   [1164297300]=>
107-   int(1164297600)
108- }
109- int(14400)
110- string(5) "test2"
111- int(7200)
112- string(4) "test"
113- Start:20061123T035500Z End:20061123T040500Z Summary:test4
114- Start:20061123T040500Z End:20061123T050500Z Summary:test3
115- Start:20061123T050500Z End:20061123T060000Z Summary:testtermin
116- Start:20061123T060000Z End:20061123T060500Z Summary:test5
117- Start:20061123T075500Z End:20061123T100500Z Summary:
118- Start:20061123T110000Z End:20061123T150000Z Summary:test2
119- Start:20061123T150000Z End:20061123T160000Z Summary:test