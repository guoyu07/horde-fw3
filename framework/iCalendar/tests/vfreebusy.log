
---- EXPECTED OUTPUT
string(9) "vFreebusy"
string(334) "BEGIN:VFREEBUSY
ORGANIZER;CN=GunnarWrobel:MAILTO:wrobel@demo2.pardus.de
DTSTAMP:20061122T230929Z
DTSTART:20061122T230000Z
DTEND:20070121T230000Z
FREEBUSY;X-UID=MmZlNWU3NDRmMGFjNjZkNjRjZjFkZmFmYTE4NGFiZTQ=;X-SUMMARY=dGVzd
 HRlcm1pbg==:20061123T050000Z/20061123T060000Z
FREEBUSY:20061123T080000Z/20061123T100000Z
END:VFREEBUSY
"
string(12) "GunnarWrobel"
string(22) "wrobel@demo2.pardus.de"
array(2) {
  [1164258000]=>
  int(1164261600)
  [1164268800]=>
  int(1164276000)
}
string(10) "testtermin"
array(1) {
  [1164261600]=>
  int(1164268800)
}
int(1164236400)
int(1169420400)
array(2) {
  [1164258000]=>
  int(1164261600)
  [1164268800]=>
  int(1164276000)
}
array(4) {
  [1164258000]=>
  int(1164261600)
  [1164268800]=>
  int(1164276000)
  [1164279600]=>
  int(1164283200)
  [1164290400]=>
  int(1164297600)
}
array(4) {
  [1164258000]=>
  array(2) {
    ["X-UID"]=>
    string(44) "MmZlNWU3NDRmMGFjNjZkNjRjZjFkZmFmYTE4NGFiZTQ="
    ["X-SUMMARY"]=>
    string(16) "dGVzdHRlcm1pbg=="
  }
  [1164268800]=>
  array(0) {
  }
  [1164279600]=>
  array(0) {
  }
  [1164290400]=>
  array(1) {
    ["X-SUMMARY"]=>
    string(8) "dGVzdA=="
  }
}
array(9) {
  [1164254100]=>
  int(1164254700)
  [1164254400]=>
  int(1164258300)
  [1164261300]=>
  int(1164261900)
  [1164268500]=>
  int(1164269100)
  [1164275700]=>
  int(1164276300)
  [1164279600]=>
  int(1164294000)
  [1164290400]=>
  int(1164290520)
  [1164292200]=>
  int(1164292500)
  [1164297300]=>
  int(1164297600)
}
int(1004297300)
int(1169420400)
array(11) {
  [1164258000]=>
  int(1164261600)
  [1164268800]=>
  int(1164276000)
  [1164279600]=>
  int(1164294000)
  [1164290400]=>
  int(1164297600)
  [1164254100]=>
  int(1164254700)
  [1164254400]=>
  int(1164258300)
  [1164261300]=>
  int(1164261900)
  [1164268500]=>
  int(1164269100)
  [1164275700]=>
  int(1164276300)
  [1164292200]=>
  int(1164292500)
  [1164297300]=>
  int(1164297600)
}
int(14400)
string(5) "test2"
int(7200)
string(4) "test"
Start:20061123T035500Z End:20061123T040500Z Summary:test4
Start:20061123T040500Z End:20061123T050500Z Summary:test3
Start:20061123T050500Z End:20061123T060000Z Summary:testtermin
Start:20061123T060000Z End:20061123T060500Z Summary:test5
Start:20061123T075500Z End:20061123T100500Z Summary:
Start:20061123T110000Z End:20061123T150000Z Summary:test2
Start:20061123T150000Z End:20061123T160000Z Summary:test
---- ACTUAL OUTPUT
string(9) "vFreebusy"

Fatal error: Call to undefined method String::wordwrap() in /home/wrobel/usr/devel/gh/horde-glue/horde-fw3/framework/iCalendar/iCalendar.php on line 1055
---- FAILED
