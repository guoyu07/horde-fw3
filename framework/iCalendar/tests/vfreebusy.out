string(9) "vFreebusy"

Fatal error: Call to undefined method String::wordwrap() in /home/wrobel/usr/devel/gh/horde-glue/horde-fw3/framework/iCalendar/iCalendar.php on line 1055